OpenDreamRuntime/Objects/DreamObjectTree.cs

```diff
@@ -473,7 +473,7 @@ public sealed class TreeEntry {
         public bool TryGetGlobalProc(string name, [NotNullWhen(true)] out DreamProc? globalProc);
         public TreeEntry GetTreeEntry(DreamPath path);
         public TreeEntry GetTreeEntry(int typeId);
-        public bool TryGetTreeEntry(DreamPath path, out TreeEntry? treeEntry);
+        public bool TryGetTreeEntry(DreamPath path, [NotNullWhen(true)] out TreeEntry? treeEntry);
```

Comment:
`NotNullWhen` is already in the method implementation. I just mirrored it here. A bit surprised that such mismatch gets no warnings during compilation.